dev/com.ibm.ws.threading/bnd.bnd

```diff
@@ -23,10 +23,12 @@ Bundle-SymbolicName: com.ibm.ws.threading; singleton:=true
 # for the exported API.
 Export-Package: com.ibm.ws.threading;provide:=true, \
                 com.ibm.wsspi.threading;provide:=true, \
-                com.ibm.ws.threading.listeners
+                com.ibm.ws.threading.listeners, \
+                io.openliberty.threading.virtual;provide:=true;-split-package:=merge-first
```

Comment:
Does this really need to be exported by the threading bundle?  Once everything is packaged into the final threading bundle I believe this package can be considered internal to the classes packaged in the threading bundle.

Can we move it to be a `Private-Package` instead?  Or are there other usages of the `io.openliberty.threading.virtual.VirtualThreadOps` interface outside of the threading bundle?

Reply:
I agree with exporting the package for VirtualThreadOps. It will be used by other components.  I don't understand why there should be a split package for it.  Can we avoid that?

Reply:
I hadn't noticed that it is used by the concurrency bundle.

Reply:
@njr-11 @tjwatson   So I added `-split-package:=merge-first` because `io.openliberty.threading.virtual` package exists both in `io.openliberty.threading.virtual` and `io.openliberty.threading.virtual.internal` bundles. 
I got a warning from the bnd plugin saying I had to add this flag.  If that is wrong let I'm happy to change it.

Reply:
Discussed this during our scrum and I think this conversation has been resolved.